### ISSUE TITLE
blank blue screen after upgrade

### ISSUE THREAD

**Issue by jan-panoch** [Intermesh/groupoffice#1188]
hello,

we freshly installed groupoffice using docker, after initial setup it did some upgrade and after upgrade i see just blank blue screen and can do nothing..

log from groupoffice docker container:

`office.*cz 172.19.0.1 - - [04/Oct/2024:10:38:49 +0000] "GET / HTTP/1.1" 200 1523
office.*.cz 172.19.0.1 - - [04/Oct/2024:10:38:49 +0000] "GET /views/Extjs3/themes/Paper/fonts/icons.woff2 HTTP/1.1" 200 99896
office.*.cz 172.19.0.1 - - [04/Oct/2024:10:38:49 +0000] "GET /views/Extjs3/themes/Paper/style.css?v=1727705197 HTTP/1.1" 200 48724
office.*.cz 172.19.0.1 - - [04/Oct/2024:10:38:49 +0000] "GET /views/Extjs3/javascript/ext-base-debug.js?mtime=1727704999 HTTP/1.1" 200 26502
office.*.cz 172.19.0.1 - - [04/Oct/2024:10:38:49 +0000] "GET /views/Extjs3/javascript/ext-all-debug.js?mtime=1727704999 HTTP/1.1" 200 280867
office.*.cz 172.19.0.1 - - [04/Oct/2024:10:38:49 +0000] "GET /views/Extjs3/themes/Paper/img/favicon/apple-touch-icon.png HTTP/1.1" 200 10789
office.*.cz 172.19.0.1 - - [04/Oct/2024:10:38:49 +0000] "GET /views/Extjs3/themes/Paper/img/favicon/favicon-16x16.png HTTP/1.1" 200 995`

 page source of the blank initial screen:

`<!DOCTYPE html>
<html lang="en" dir="ltr">
<head>
		<meta charset="UTF-8">
	<meta name="viewport" content="width=device-width, initial-scale=1">
	<meta name="HandheldFriendly" content="true">
	<meta http-equiv="x-ua-compatible" content="IE=edge">
	<meta name="apple-mobile-web-app-capable" content="no">
	<meta name="apple-mobile-web-app-title" content="Group-Office">
	<meta name="apple-mobile-web-app-status-bar-style" content="black">
    <meta name="theme-color" content="rgb(22, 82, 161)">
	<meta name="msapplication-TileColor" content="rgb(22, 82, 161)">

    <link rel="icon" type="image/png" sizes="32x32" href="/views/Extjs3/themes/Paper/img/favicon/favicon-32x32.png">
    <link rel="icon" type="image/png" sizes="16x16" href="/views/Extjs3/themes/Paper/img/favicon/favicon-16x16.png">
    <link rel="apple-touch-icon" sizes="180x180" href="/views/Extjs3/themes/Paper/img/favicon/apple-touch-icon.png">
    <link rel="manifest" href="/views/Extjs3/themes/Paper/img/favicon/site.webmanifest">
    <link rel="mask-icon" href="/views/Extjs3/themes/Paper/img/favicon/safari-pinned-tab.svg" color="#888888">
    <title>Group-Office</title>

    <link rel="preload" href="/views/Extjs3/themes/Paper/fonts/icons.woff2" as="font" type="font/woff2" crossorigin>
    <link rel="stylesheet" media="print, screen"  href="/views/Extjs3/themes/Paper/style.css?v=1727705197">
<script type="text/javascript" src="/views/Extjs3/javascript/ext-base-debug.js?mtime=1727704999"></script><script type="text/javascript" src="/views/Extjs3/javascript/ext-all-debug.js?mtime=1727704999"></script><script type="text/javascript" src="http://office.*.cz/views/Extjs3/lang.php?lang=en&v=1728038315"></script>

<script type="text/javascript">

	Ext.namespace("GO");

	GO.settings = {"max_rows_list":50,"config":{"theme":"Paper","product_name":"GroupOffice","host":"\/","title":"Group-Office","full_url":"http:\/\/office.*.cz\/","allow_password_change":true,"allow_themes":true,"allow_profile_edit":true,"max_users":0,"debug":false,"max_attachment_size":52428800,"max_file_size":1048576000,"help_link":null,"support_link":"https:\/\/www.group-office.com\/support\/","report_bug_link":"http:\/\/sourceforge.net\/tracker\/?group_id=76359&atid=547651","nav_page_size":50,"logoutWhenInactive":0,"tickets_no_email_required":false,"default_country":null,"checker_interval":120,"remember_login":true,"encode_callto_link":false,"login_message":null,"hideAbout":null,"email_allow_body_search":false,"lostPasswordURL":null},"state_index":"go","language":"en","show_contact_cf_tabs":[]};
	GO.language = "en";
	GO.calltoTemplate = 'tel://{phone}';
	GO.calltoOpenWindow = false;
GO.securityToken="Us6tEdhav5IxuSOYF1G7";window.name="e353e6bef5319efd9bd8a80e38741266";Ext.Ajax.extraParams={security_token:"Us6tEdhav5IxuSOYF1G7"};</script>
<script type="text/javascript" src="http://office.*.cz/views/Extjs3/script.php?v=1728022766"></script><script type="text/javascript">

Ext.onReady(GO.mainLayout.boot, GO.mainLayout);
</script>

	<script type="text/javascript">
		GO.util.density = GO.util.isMobileOrTablet() ? 160 :  140;
	</script>
        <link rel="stylesheet" href="http://office.*.cz/views/Extjs3/css.php?theme=/views/Extjs3/themes/Paper/&v=1728022766"  />

    <link rel="stylesheet" media="screen" href="http://office.*.cz/views/goui/dist/goui/style/groupoffice.css" />

<style>
    :root, body{
            }
    body.dark{
            }
</style>
</head>
<body class="">
<div id="paper"></div>`

compose.yaml

`services:
  groupoffice:
    image: intermesh/groupoffice
    restart: always
    ports:
      - "9000:80"
    links:
      - db
    volumes:
      - "/data/office.*.cz/godata:/var/lib/groupoffice:cached"
      - "/data/office.*.cz/goetc:/etc/groupoffice:cached"
#      - "gostudio:/usr/local/share/groupoffice/go/modules/studio:cached"
      - ./src/studio:/usr/local/share/groupoffice/go/modules/studio

    #      Example for ssl
#      - "./config/ssl/groupoffice:/etc/ssl/groupoffice"

    env_file:
      - ./db.env
  db:
    image: mariadb:11.1.2
    restart: always
    env_file:
      - ./db.env
    environment:
      MYSQL_ROOT_PASSWORD: *
      MARIADB_AUTO_UPGRADE: 1
    volumes:
      - "/data/office.*.cz/dbdata:/var/lib/mysql:cached"
volumes:
  godata:
  goetc:
  #gostudio:
  dbdata:`

logfile produced by upgrade:

`cat 20241004_081922.log`
`Rebuilding cache`
`Registering all entities`
`Done!`

real domain name replaced by *

any suggestions?

thank you 

Jan

**Comment by derjoachim** [comment]
There's no server side error being thrown, so I can only guess. Are you getting a client side error by any chance? You can check the developer panel (Ctrl+Shift+i) , tab 'console' for errors.

**Comment by derjoachim** [comment]
Closing this ticket due to inactivity.
